### PR TITLE
Add ca-certificates to image

### PR DESCRIPTION
**Pull request by j16r**
[Fixes #159012037]

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,6 +14,9 @@ RUN make build
 # Final layer: ubuntu with chainlink binary
 FROM ubuntu:16.04
```
Comment (dimroc) [inline Dockerfile]: Unrelated to PR. Our final image size has ballooned to ~370MB. Should we use `alpine` linux instead to get smaller images?

**Comment by j16r** [comment]
I think only the SGX image is that big?
https://hub.docker.com/r/smartcontract/chainlink/tags/ 86mb for acceptance
and 50mb for latest

But right now, no, we cannot use alpine because of the differences between
linuxes and the way Go does linking. We'd have to make the base image
alpine as well.

**Comment by dimroc** [comment]
What's wrong with making our base image and/or builder alpine? Just curious.

**Comment by dimroc** [comment]
Perhaps we need ubuntu for SGX.

**Comment by j16r** [comment]
@dimroc we totally should try and build an alpine base image, but we should look into porting the SGX stuff over at the same time as I'd prefer not to have multiple build images to maintain.

**Comment by j16r** [comment]
See http://acceptance-node.chain.link/job_specs/3efa0226002344789f92658251b824f9

**Pull request merged**